server/routes/offenceCodeDecisions/decisionForm.ts

```diff
@@ -0,0 +1,20 @@
+// Form data backing a decision. Used in form submits and to allow the saving of form data on a decision page so that
+// we can repopulate it when returning from a user lookup.
+export type DecisionForm = {
```

Comment:
This is the data structure that backs a decision page. We use it to retain information in the session when we go off to lookup e.g. a prisoner, or we bind it on a post request - its basically a form backing object